src/Test.FunctionalTests.BinSkim.Rules/RuleTests.cs

```diff
@@ -277,13 +278,23 @@ private BinaryAnalyzerContext CreateContext(TestMessageLogger logger, Properties
                 context.Rule = skimmer;
 
                 AnalysisApplicability applicability = skimmer.CanAnalyze(context, out string reasonForNotAnalyzing);
+
                 if (applicability != expectedApplicability)
                 {
                     sb.AppendLine("CanAnalyze did not correct indicate target applicability (unexpected return was " +
```

Comment:
>correct [](http://example.com/codeflow?start=54&length=7)

s/be 'correctly'. You didn't make this mistake but would you mind correcting it?

Reply:
btw - as you resolve things, you can mark them as 'resolved' in code flow. Then your reviewers can 'close' them. :)